homeassistant/components/stookwijzer/config_flow.py

```diff
@@ -0,0 +1,46 @@
+"""Config flow to configure the Stookwijzer integration."""
+from __future__ import annotations
+
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_LATITUDE, CONF_LOCATION, CONF_LONGITUDE
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.selector import LocationSelector
+
+from .const import DOMAIN
+
+
+class StookwijzerFlowHandler(ConfigFlow, domain=DOMAIN):
+    """Config flow for Stookwijzer."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle a flow initialized by the user."""
+
+        if user_input is not None:
+            lat: float = user_input[CONF_LOCATION][CONF_LATITUDE]
+            lon: float = user_input[CONF_LOCATION][CONF_LONGITUDE]
+
+            await self.async_set_unique_id(f"{lat}-{lon}")
+            self._abort_if_unique_id_configured()
```

Comment:
Setting unique IDs on lat/long is IMHO useless, it can easily be shifted just a tiny bit.

We can just as well just not do it.

```suggestion
```